entrypoint.sh

```diff
@@ -27,7 +27,3 @@ unset JAVA_HOME
 
 sonar-scanner -Dsonar.projectBaseDir=${INPUT_PROJECTBASEDIR} ${INPUT_ARGS}
 
-_tmp_file=$(ls "${INPUT_PROJECTBASEDIR}/" | head -1)
-PERM=$(stat -c "%u:%g" "${INPUT_PROJECTBASEDIR}/$_tmp_file")
-
-chown -R $PERM "${INPUT_PROJECTBASEDIR}/"
```

Comment:
This will fail the tests, because right now it doesn't take into account the fact it should run a "post entrypoint" script. Could you add the following line [after line 93 here](https://github.com/SonarSource/sonarqube-scan-action/blob/master/test/run-qa.sh#L93) (i.e., after the existing `docker run` command):

```bash
docker run -v `pwd`:/github/workspace/ --workdir /github/workspace --network $network --env INPUT_PROJECTBASEDIR=/github/workspace --entrypoint /cleanup.sh sonarsource/sonarqube-scan-action
```